src/Files.App/Actions/Show/ToggleSidebarAction.cs

```diff
@@ -0,0 +1,41 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Extensions;
+using Files.App.ViewModels;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class ToggleSidebarAction : ObservableObject, IToggleAction
+	{
+		private readonly SidebarViewModel viewModel = Ioc.Default.GetRequiredService<SidebarViewModel>();
```

Comment:
I don't think we should add a reference to the SidebarViewModel, perhaps a service or helper would be more appropriate.

Reply:
It's not a blocker, we can certainly review the different actions and refactor these types or references a different time.

Reply:
I'll think about a solution

Reply:
I think we can leave it as is